src/Compilers/CSharp/Portable/Emitter/Model/NamespaceSymbolAdapter.cs

```diff
@@ -16,17 +16,10 @@ internal partial class
 #endif 
         Cci.INamespace
     {
-#if DEBUG
-        internal NamespaceSymbolAdapter(NamespaceSymbol underlyingNamespaceSymbol)
-        {
-            AdaptedNamespaceSymbol = underlyingNamespaceSymbol;
-        }
+        Cci.INamespace Cci.INamespace.ContainingNamespace => AdaptedNamespaceSymbol.ContainingNamespace?.GetCciAdapter();
+        string Cci.INamedEntity.Name => AdaptedNamespaceSymbol.MetadataName;
```

Comment:
Nit: please either put a newline between the above properties, or remove the one below.